README.md

```diff
@@ -1,22 +1,97 @@
-# rhoai-RAG-pattern
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern">
+    <img src="images/Logo_RAG_LLM_Pattern.png" alt="Logo" width="160" height="160">
+  </a>
 
-Today a dominant usecase for deploying Large Language Models (LLMs) within the enteprise is as a enterprise search assistant. LLMs without any underlying context a prone to halucination. Retrieval Augmented Generation (RAG) is an architecture where the LLM is pared with a vector database. The vector database contains a specialised index of enterprise documents which provide 'context' to the LLM allowing more accurate results AND allowing employees to follow up with source documents. This project is a reference pattern for deploying a RAG search assistant built on top of Red Hat OpenShift AI.
+  <h3 align="center">Validated Pattern for Retrieval-Augmented Generation (RAG) with LLM</h3>
+  
+  <p align="center">
+    <br />
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern">View Demo</a>
+    ·
+    <a href="https://github.com/opendatahub-io-contrib/genai-llm-rag-pattern/issues/new?labels=bug&template=bug-report---.md">Report Bug</a>
```

Comment:
Templates don't exist yet for bug / feature requests.

Reply:
Use the main link until we create suitable templates